ts/src/woofipro.ts

```diff
@@ -0,0 +1,2636 @@
+        // }
+        //
+        const extendParams = { 'symbol': symbol };
+        if (isByClientOrder) {
+            extendParams['client_order_id'] = clientOrderIdExchangeSpecific;
+        } else {
+            extendParams['id'] = id;
+        }
+        if (stop) {
+            return this.extend (this.parseOrder (response), extendParams);
+        }
+        const data = this.safeDict (response, 'data', {});
+        return this.extend (this.parseOrder (data), extendParams);
+    }
+
+    async cancelOrders (ids:string[], symbol: Str = undefined, params = {}) {
+        /**
+         * @method
+         * @name woofipro#cancelOrders
+         * @description cancel multiple orders
+         * @see https://orderly.network/docs/build-on-evm/evm-api/restful-api/private/batch-cancel-orders
+         * @see https://orderly.network/docs/build-on-evm/evm-api/restful-api/private/batch-cancel-orders-by-client_order_id
+         * @param {string[]} ids order ids
+         * @param {string} [symbol] unified market symbol
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {string[]} [params.client_order_ids] max length 10 e.g. ["my_id_1","my_id_2"], encode the double quotes. No space after comma
+         * @returns {object} an list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+         */
+        await this.loadMarkets ();
+        const clientOrderIds = this.safeListN (params, [ 'clOrdIDs', 'clientOrderIds', 'client_order_ids' ]);
+        params = this.omit (params, [ 'clOrdIDs', 'clientOrderIds', 'client_order_ids' ]);
+        const request = {};
+        let response = undefined;
+        if (clientOrderIds) {
+            request['client_order_ids'] = clientOrderIds.join (',');
+            response = await this.v1PrivateDeleteClientBatchOrder (this.extend (request, params));
+        } else {
+            request['order_ids'] = ids.join (',');
+            response = await this.v1PrivateDeleteBatchOrder (this.extend (request, params));
+        }
+        //
+        // {
+        //     "success": true,
+        //     "timestamp": 1702989203989,
+        //     "data": {
+        //         "status": "CANCEL_ALL_SENT"
+        //     }
+        // }
+        //
+        return response;
+    }
+
+    async cancelAllOrders (symbol: Str = undefined, params = {}) {
+        /**
+         * @method
+         * @name woofipro#cancelAllOrders
+         * @see https://orderly.network/docs/build-on-evm/evm-api/restful-api/private/cancel-all-pending-algo-orders
+         * @see https://orderly.network/docs/build-on-evm/evm-api/restful-api/private/cancel-orders-in-bulk
+         * @description cancel all open orders in a market
+         * @param {string} symbol unified market symbol
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {boolean} [params.stop] whether the order is a stop/algo order
```

Comment:
the unified parameter is `trigger`, `stop` is deprecated we should still read in params but not place it in the docs